python_frank_energie/frank_energie.py

```
@@ -20,9 +20,10 @@
 from .exceptions import (AuthException, AuthRequiredException,
                          FrankEnergieException, NetworkError,
                          RequestException)
-from .models import (Authentication, EnergyConsumption, EnodeChargers, EnodeVehicles, Invoices,
+from .models import (Authentication, EnergyConsumption, EnodeChargers, EnodeVehicle, EnodeVehicles, Invoices,
```

Flake8 (7.3.0)

[error] redefinition of unused 'Authentication' from line 19

(F811)